app/src/main/resources/assembly/assembly.xml

```diff
@@ -0,0 +1,39 @@
+<assembly>
+    <id>eip-pro-client</id>
+    <formats>
+        <format>tar</format>
+        <format>zip</format>
+    </formats>
+    <fileSets>
+        <fileSet>
+            <directory>src/main/resources/scripts</directory>
+            <outputDirectory>.</outputDirectory>
+            <includes>
+                <include>start.sh</include>
+                <include>start.bat</include>
```

Comment:
General remark: I think we have both `start.bat` and `start.cmd` for the Windows file in different repos. We should stick to one (I have no preference).

Reply:
Then we should stick with `.cmd`, they are modern batch files(using in Windows NT and later versions) which offers more capabilities, additional commands, better scripting support, and enhanced error handling compared to `.bat` files.